### PR TITLE
Igvf 90 donor phenotypic feature

### PR DESCRIPTION
**Pull request by KAndreeva**
Removed health_status_history from human donor. Added phenotypic_feature to Donor.

### CODE REVIEW

File: src/igvfd/schemas/changelogs/donor.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 * Add `description`.
 * Add `revoke_detail`.
+* Add `phenotypic_feature`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/donor.md:7]: should be plural after last week discussion
phenotypic_features

File: src/igvfd/schemas/changelogs/human_donor.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`human_donor.json`*
 
+### Schema version 3
+
+* Remove `health_status_history`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/human_donor.md:5]: 1. remove health_status_history
2. add phenotypic_features

File: src/igvfd/schemas/donor.json [hunk 3]
```diff
@@ -81,6 +81,20 @@
                 "linkTo": "PhenotypeTerm"
             }
         },
+        "phenotypic_feature": {
```
Comment (gabdank) [inline src/igvfd/schemas/donor.json:84]: should be plural after last week discussion
phenotypic_features

File: src/igvfd/schemas/donor.json [hunk 4]
```diff
@@ -81,6 +81,20 @@
...
+            "title": "Phenotypic Features",
+            "description": "A list of assocated phenotypic features of the donor",
```
Comment (gabdank) [inline src/igvfd/schemas/donor.json:86]: assocated -> associated

File: src/igvfd/tests/data/inserts/rodent_donor.json [hunk 5]
```diff
@@ -24,6 +24,9 @@
             "/phenotype-terms/1b267ef2-f42e-11ec-b939-0242ac120002/",
             "/phenotype-terms/HP_0001658/"
         ],
+        "phenotypic_feature": [
```
Comment (gabdank) [inline src/igvfd/tests/data/inserts/rodent_donor.json:27]: plural

File: src/igvfd/tests/data/inserts/rodent_donor.json [hunk 6]
```diff
@@ -61,6 +64,10 @@
         "parents": [
             "c37934b0-4269-4470-be53-9eac7b196447"
         ],
+        "phenotypic_feature": [
```
Comment (gabdank) [inline src/igvfd/tests/data/inserts/rodent_donor.json:67]: plural

**Review by gabdank — CHANGES_REQUESTED** [review]
see PR comments

File: src/igvfd/schemas/changelogs/donor.md (hunk 1 above)
Comment (KAndreeva) [reply src/igvfd/schemas/changelogs/donor.md]: Done

File: src/igvfd/schemas/changelogs/human_donor.md (hunk 2 above)
Comment (KAndreeva) [reply src/igvfd/schemas/changelogs/human_donor.md:5]: Done

File: src/igvfd/schemas/donor.json (hunk 3 above)
Comment (KAndreeva) [reply src/igvfd/schemas/donor.json]: Done

File: src/igvfd/schemas/donor.json (hunk 4 above)
Comment (KAndreeva) [reply src/igvfd/schemas/donor.json]: Done

File: src/igvfd/tests/data/inserts/rodent_donor.json (hunk 5 above)
Comment (KAndreeva) [reply src/igvfd/tests/data/inserts/rodent_donor.json]: Done

File: src/igvfd/tests/data/inserts/rodent_donor.json (hunk 6 above)
Comment (KAndreeva) [reply src/igvfd/tests/data/inserts/rodent_donor.json]: Done

**Review by gabdank — APPROVED** [review]

**Pull request merged**